Application/EdFi.Ods.Sandbox/Provisioners/PostgresSandboxProvisioner.cs

```diff
@@ -10,6 +10,7 @@
 using EdFi.Admin.DataAccess.Utils;
 using EdFi.Ods.Common.Configuration;
 using log4net;
+using Microsoft.Extensions.Configuration;
```

Comment:
same problem as above, needed to make a copy and updated.

Reply:
Created new file and wrapped inside NETFRAMEWORK

Reply:
as a fyi in the future, we would normally create a new file for the netcoreapp changes. and modify the original file to be netframework.